src/CattyTests/Bricks/SetBackgroundBrickTests.swift

```diff
@@ -63,9 +61,9 @@ final class SetBackgroundBrickTests: AbstractBrickTest {
         var look1: Look!
         do {
             look = Look(name: "test", andPath: "test.png")
-            try imageData?.write(to: URL(fileURLWithPath: object.projectPath() + "images/test.png"))
+            try imageData?.write(to: URL(fileURLWithPath: object.projectPath() + "Scene 1/images/test.png"))
```

Comment:
Please do not use the hardcoded name of the scene here. I would suggest to use `project.scene.imagesPath` instead (or similar).